packages/node-image-compute/base.packages

```diff
@@ -1,9 +1,110 @@
-# Packages common to all mediums.
+# Packages for any CN artifacts.
 # Format:
 #   package_name=version
 # The version is the same version reported by the OS package manager (e.g. zypper).
-cfs-state-reporter=1.9.0-1
-cfs-trust=1.6.0-1
-cray-switchboard=1.2.3-1
-cray-uai-util=1.0.13-1
-craycli=0.64.0-1
+
+# Analytics Base SLES Packages
+java-11-openjdk-devel
+
+# CDST GUI Packages
+libGLU1
+libatk-1_0-0
+libcairo-gobject2
+libcairo2
+libgcrypt20
+libgdk_pixbuf-2_0-0
+libgio-2_0-0
+libglvnd
+# libgpg-error # 'libgpg-error0' providing 'libgpg-error' is already installed.
+libgtk-3-0
+libgtksourceview-3_0-1
+libopenblas_pthreads0
+# librsvg2 # 'librsvg-2-2' providing 'librsvg2' is already installed.
+munge
+typelib-1_0-Gtk-3_0
+
+# COS General Packages
+cray-atom-energy-plugin
+cray-boot-parameters-shasta-compute
+cray-bpmcdmod
+#cray-bpmcdmod-kmp-cray_shasta_c
+cray-chrony-dracut
+cray-cps-dracut
+#cray-craytrace-kmp-cray_shasta_c
+#cray-freemem-kmp-cray_shasta_c
+cray-hugepage-setup
+cray-liborca
+cray-libxpmem-devel
+cray-libxpmem0
+cray-netif-dracut
+cray-network-dracut
+cray-node-identity
+cray-orca
+cray-overlay-preload
+cray-power-management
+#cray-power-management-kmp-cray_shasta_c
+cray-spire-dracut
+cray-system-files
+cray-system-setup-scripts
+cray-systemd-presets
+cray-udev-network-cn
+cray-xpmem
+#cray-xpmem-kmp-cray_shasta_c
+#kernel-cray_shasta_c
+#kernel-cray_shasta_c-devel
+
+# COS SHASTA-OS packages
+cray-ckdump-helper
+cray-cos-release
+cray-cps-utils
+cray-crash
+cray-crash-extensions
+cray-low-noise-mode
+kdump-spire-watcher
+
+# COS SHASTA-3RD Packages
+acpid
+#amd-energy-kmp-cray_shasta_c
+#amd-hsmp-kmp-cray_shasta_c
+cray-hwloc
+cray-libhugetlbfs
+cray-libhugetlbfs-devel
+cray-makedumpfile
+cray-rasdaemon
+esmi-ib-library
+libs3
```

Comment:
potentially used by ckdump
in general it is useful for talking to S3, might move to node-images-base